pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -350,7 +350,7 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration, featur
 	}
 
 	if kc.EnableSystemLogQuery && !localFeatureGate.Enabled(features.NodeLogQuery) {
-		allErrors = append(allErrors, fmt.Errorf("invalid configuration: NodeLogQuery feature gate is required for enableSystemLogHandler"))
+		allErrors = append(allErrors, fmt.Errorf("invalid configuration: NodeLogQuery feature gate is required for enableSystemLogQuery"))
```

Comment:
wither message is correct. As both must be specified. Just for keeping code consistent:

/lgtm
/approve

Reply:
@SergeyKanzhelev, thanks for reviewing. Are you asking for a change?